discord/ext/commands/help.py

```diff
@@ -995,10 +995,13 @@ def add_indented_commands(self, commands, *, heading, max_size=None):
         max_size = max_size or self.get_max_size(commands)
 
         get_width = discord.utils._string_width
-        for command in commands:
-            name = command.name
-            width = max_size - (get_width(name) - len(name))
-            entry = f'{self.indent * " "}{name:<{width}} {command.short_doc}'
+        last_name = ""
+        for command_name, command in [(command.name, command) for command in commands]:
+            if last_name == command_name:
+                continue
```

Comment:
There is an edge case of subcommands not registering (e.g. "foo bar" and "goo bar").